### PR TITLE
Add an initial checklist to the performance troubleshooting page

### PR DESCRIPTION
**Pull request by peppy**
Had multiple cases where users are using weird skins which cause issues that they blame on the game. Adding this as a prominent point may help.  Also added a few other common ones.

### CODE REVIEW

File: wiki/Performance_troubleshooting/en.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ osu! does its best to run with sane default settings and proactively alert you i
 
 Unfortunately, due to the scope of hardware which osu! runs on, it's very hard to ensure optimal performance for every single user. This page aims to help you self-diagnose performance issues you have, but also provides specific terms you can learn (and checks you can perform) to help us help you when you reach out for support.
 
+## Things to check
+
+- Are you using a skin? If so, try switching to the default and see if things improve. Some skins can be crafted in a way which can cause performance issues on some or all systems.
+- Is your frame limiter set to "unlimited"? Set it to "optimal" and see if it helps. Read down for more information on why "unlimited" is usually not the best choice.
```
Comment (Walavouchey) [inline wiki/Performance_troubleshooting/en.md:10]:
```suggestion
- Is your frame limiter set to "unlimited"? Set it to "optimal" and see if it helps. See [its section below](#unlimited) for more information on why "unlimited" is usually not the best choice.
```

**Comment by Walavouchey** [comment]
i take it that this is intended to supersede https://github.com/ppy/osu-wiki/pull/6765?

**Comment by peppy** [comment]
Ah.. I didn't see that. But it may be that the other PR was also spurred by a stupid twitter conversation that happened of recent since it covers similar points.

Not sure about the fullscreen optimisations part mentioned in that PR, need reference to that actually helping.

Also wanted to put these points at the top of the page for people that have short attention spans.

**Review by Gamingzone123 — APPROVED** [review]
looks good to me

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**